scripts/build/babel-plugins/transform-eval-require.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+
+//
+// BEFORE:
+//   __r("path/to/file")
```

Comment:
Since this isn't transforming `eval("require")` but now `__r`, we could rename the plugin.